terraform/typescript.yaml

```diff
@@ -2,6 +2,7 @@
 chloechien: 'maintainer'
 verbanicm: 'maintainer'
 mattrandallbecker: 'maintainer'
+ruizhg: 'maintainer'
```

Comment:
Can you move this below line 8 and change to `member` instead of `maintainer`?


```
ruizhg: 'member'
```

Reply:
Updated